ui/src/lib/components/k8s/DataTable/component.svelte

```diff
@@ -250,9 +250,9 @@
                         {/each}
                       </ul>
                     {:else if modifier === 'link-external'}
-                      <Link href={value.href || value} text={value.text || value} target={'_blank'} />
+                      <Link href={value.href || ''} text={value.text || value} target={'_blank'} />
```

Comment:
```suggestion
                      <Link href={value.href || ''} text={value.text || ''} target={'_blank'} />
```
might be wrong on this, but i think if we set the href to an empty string instead of value then we're eliminating the possibility to ever send a simple value as the subject of both href and text (e.g. `value = example.com`). so we're enforcing that the data passed to Link must arrive in the shape of `{href: '',text: ''}` if an actual link is to be created.

Reply:
👍 discussed sync